c7n/resources/connect.py

```diff
@@ -58,3 +59,53 @@ def process(self, resources, event=None):
                 results.append(r)
 
         return results
+
+    @Connect.action_registry.register("set-attributes")
+    class SetAttributes(Action):
+        """Set the attributes for the connect resources
+
+        :Example:
+
+        .. code-block:: yaml
+
+            policies:
+              - name: connect-set-contact-lens
+                resource: connect-instance
+                filters:
+                  - type: instance-attribute
+                    key: Attribute.Value
+                    value: false
+                    attribute_type: CONTACT_LENS
+                actions:
+                  - type: set-attributes
+                    attribute_type: CONTACT_LENS
+                    value: true
+              - name: connect-disable-contact-lens
+                resource: connect-instance
+                filters:
+                  - type: instance-attribute
+                    key: Attribute.Value
+                    value: true
+                    attribute_type: CONTACT_LENS
+                actions:
+                  - type: set-attributes
+                    attribute_type: CONTACT_LENS
+                    value: false
+        """
+        attributes = ["INBOUND_CALLS", "OUTBOUND_CALLS",
+                      "CONTACTFLOW_LOGS", "CONTACT_LENS",
+                      "AUTO_RESOLVE_BEST_VOICES", "USE_CUSTOM_TTS_VOICES",
+                      "EARLY_MEDIA", "MULTI_PARTY_CONFERENCE",
+                      "HIGH_VOLUME_OUTBOUND", "ENHANCED_CONTACT_MONITORING"]
+        schema = type_schema("set-attributes", attribute_type={'anyOf': [{'enum': attributes},
+                  {'type': 'string'}]}, value={}, required=["value", "attribute_type"])
+        permissions = ("connect:UpdateInstanceAttribute",)
+
+        def process(self, resources):
+            client = local_session(self.manager.session_factory).client('connect')
+
+            for r in resources:
+                client.update_instance_attribute(InstanceId=r["Id"],
+                    AttributeType=self.data.get("attribute_type"), Value=self.data.get("value"))
+
+            return
```

Comment:
I don't think this return statement is needed since we want to bubble up the exception (if any) to the user.

Reply:
exceptions will bubble up anyway as there's no try/except here, the return in general is not needed for actions